### ISSUE TITLE
ImportError: No module named pbr.version

### ISSUE THREAD

**Issue by grzn** [pycontribs/jira#510]
```
pip install jira
Collecting jira
  Downloading https://pypi.infinidat.com/media/dists/jira-1.0.11-py2.py3-none-any.whl (95kB)
    100% |████████████████████████████████| 102kB 8.1MB/s
Collecting defusedxml (from jira)
  Using cached https://pypi.infinidat.com/media/dists/defusedxml-0.5.0-py2.py3-none-any.whl
Collecting six>=1.10.0 (from jira)
  Using cached https://pypi.infinidat.com/media/dists/six-1.11.0-py2.py3-none-any.whl
Collecting ordereddict; python_version < "3.1" (from jira)
Collecting requests>=2.10.0 (from jira)
  Using cached https://pypi.infinidat.com/media/dists/requests-2.18.4-py2.py3-none-any.whl
Requirement already satisfied: setuptools>=20.10.1 in /Users/guy/.virtualenvs/jira/lib/python2.7/site-packages (from jira)
Collecting argparse; python_version < "3.2" (from jira)
  Using cached https://pypi.infinidat.com/media/dists/argparse-1.4.0-py2.py3-none-any.whl
Collecting requests-oauthlib>=0.6.1 (from jira)
  Using cached https://pypi.infinidat.com/media/dists/requests_oauthlib-0.8.0-py2.py3-none-any.whl
Collecting requests-toolbelt (from jira)
  Using cached https://pypi.infinidat.com/media/dists/requests_toolbelt-0.8.0-py2.py3-none-any.whl
Collecting idna<2.7,>=2.5 (from requests>=2.10.0->jira)
  Using cached https://pypi.infinidat.com/media/dists/idna-2.6-py2.py3-none-any.whl
Collecting urllib3<1.23,>=1.21.1 (from requests>=2.10.0->jira)
  Using cached https://pypi.infinidat.com/media/dists/urllib3-1.22-py2.py3-none-any.whl
Collecting certifi>=2017.4.17 (from requests>=2.10.0->jira)
  Downloading https://pypi.infinidat.com/media/dists/certifi-2018.1.18-py2.py3-none-any.whl (151kB)
    100% |████████████████████████████████| 153kB 5.0MB/s
Collecting chardet<3.1.0,>=3.0.2 (from requests>=2.10.0->jira)
  Using cached https://pypi.infinidat.com/media/dists/chardet-3.0.4-py2.py3-none-any.whl
Collecting oauthlib>=0.6.2 (from requests-oauthlib>=0.6.1->jira)
Installing collected packages: defusedxml, six, ordereddict, idna, urllib3, certifi, chardet, requests, argparse, oauthlib, requests-oauthlib, requests-toolbelt, jira
Successfully installed argparse-1.4.0 certifi-2018.1.18 chardet-3.0.4 defusedxml-0.5.0 idna-2.6 jira-1.0.11 oauthlib-2.0.6 ordereddict-1.1 requests-2.18.4 requests-oauthlib-0.8.0 requests-toolbelt-0.8.0 six-1.11.0 urllib3-1.22
⮀ python
Python 2.7.14 (default, Feb  6 2018, 20:04:00)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import jira
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/guy/.virtualenvs/jira/lib/python2.7/site-packages/jira/__init__.py", line 4, in <module>
    from pbr.version import VersionInfo
ImportError: No module named pbr.version
```

Note: pypi.infinidat.com is our on-premise PyPI server, and as you can see it has the latest version.

Manually installing _pbr_ obviously solves it, but this not really a solution.

**Comment by Stealthii** [comment]
#501 is already tracking this issue. It was fixed in PR #504, which has not yet been released.

**Comment by hdost** [comment]
Closing to keep tracking in #501
